openapi-cli/src/main/java/io/ballerina/ballerina/OpenApiConverterUtils.java

```diff
@@ -307,11 +307,13 @@ private static String checkAvailabilityOfGivenName(String openApiName, File[] li
 
         for (File file : listFiles) {
             if (file.getName().equals(openApiName)) {
-                String userInput = System.console().readLine("There is already a/an " + file.getName() +
-                        " in the location. Do you want to override the file? [y/N] ");
-                if (!Objects.equals(userInput.toLowerCase(Locale.ENGLISH), "y")) {
-                    int duplicateCount = 0;
-                    openApiName = setGeneratedFileName(listFiles, openApiName, duplicateCount, isJson);
+                if (System.console() != null) {
```

Comment:
Will this always prompt the user for an input?? Can't we pass the `y` flag to the command alternatively??